acestep/gradio_ui/events/__init__.py

```
@@ -531,32 +534,61 @@ def setup_event_handlers(demo, dit_handler, llm_handler, dataset_handler, datase
-                results_section["lm_metadata_state"]
+                results_section["lm_metadata_state"],
+                generation_section["lyrics"],
+                generation_section["captions"],
             ],
             outputs=[
                 generation_section["src_audio"],
                 generation_section["generation_mode"],
                 generation_section["lyrics"],
                 generation_section["captions"],
-            ]
+            ] + _mode_ui_outputs,
```

Ruff (0.15.0)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)